package.json

```diff
@@ -23,6 +24,7 @@
     "mutexify": "^1.2.0"
   },
   "devDependencies": {
+    "@types/node": "^11.12.1",
```

Comment:
If you are using node core libraries in a Typescript project you need `@types/node` otherwise nothing will build. But given in this case it is required as a _dev_ dependency, its inclusion is entirely harmless, since it is only being used during development and testing*. End-users can avoid install deve dependencies by installing a production build.

Really an npm module with pretensions of taking its place in the modern node ecosystem should have typings as a matter of course.

* unless something in the dependency tree is using it as a mainline dependency, in which case it's going along for the ride willy-nilly. But it's only a few hundred K and you won't see hide nor hair of it in the final transpiled product. It imposes absolutely no runtime penalty whatever.

Reply:
Agreed. this looks good.